src/game.cpp

```diff
@@ -4545,8 +4561,11 @@ void Game::showPauseMenu()
 	if (!simple_singleplayer_mode) {
 		os << "button_exit[4," << (ypos++) << ";3,0.5;btn_change_password;"
 			<< strgettext("Change Password") << "]";
+	} else {
+		os << "textarea[4.95,0;3.5,6;;" << strgettext("Game Paused") << ";]";
```

Comment:
Don't we have a version of `strgettext` for formspecs (or is that only for Lua?)? If we do, it would be best to use it instead, as it could break the formspec if for some reason special characters are included in the translation (I'm mainly thinking about `,` and `;` here).